packages/build/src/packaging/download-crypt-library.ts

```diff
@@ -13,8 +13,8 @@ export async function downloadCryptLibrary(variant: PackageVariant | 'host'): Pr
   console.info('mongosh: downloading latest crypt shared library for inclusion in package:', JSON.stringify(opts));
 
   const cryptTmpTargetDir = path.resolve(__dirname, '..', '..', '..', '..', 'tmp', 'crypt-store', variant);
-  // Download mongodb for latest server version.
-  const libdir = await downloadMongoDb(cryptTmpTargetDir, 'stable', opts);
+  // Download mongodb for latest server version, including rapid releases.
+  const libdir = await downloadMongoDb(cryptTmpTargetDir, 'continuous', opts);
```

Comment:
Asked on Slack to check whether this seemed like a reasonable thing to do, will wait for an answer before moving this out of draft state